src/lib/queues.ts

```diff
@@ -35,7 +35,7 @@ export class BullQueue implements IQueue {
   }
 }
 
-export const getMoveContactQueue = () => new BullQueue(
+export const getChtConfQueue = () => new BullQueue(
   WorkerConfig.moveContactQueue, 
```

Comment:
suggestion: should we rename `WorkerConfig.moveContactQueue` to `WorkerConfig.chtConfQueue` to align with the refactored generalization ?

Reply:
yep! renamed to `chtConfQueueName`